Practise_ClassWork/Fridge.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Practise_ClassWork
+{
+    public delegate void FridgeDelegate(FridgeEvent e);
+
+    public class Fridge
+    {
+        private States state;
+
+        public event FridgeDelegate EventFridge;
+
+        protected virtual void OnFridgeState(FridgeEvent e)
+        {
+            this.EventFridge?.Invoke(e);
+        }
+
+        public void OpenFridge()
+        {
+            this.state = States.Opened_fridge;
+            Console.WriteLine("Fridge is opened");
```

Comment:
We should have used the event approach to notify the Human about was really happened.

Reply:
-4 to logic